packages/frontend-2/composables/logging.ts

```diff
@@ -1,9 +1,10 @@
 import { NuxtApp } from '#app'
 import { Optional } from '@speckle/shared'
 import { buildFakePinoLogger } from '~~/lib/core/helpers/observability'
+import type pino from 'pino'
 
 export const useLogger = () => {
-  return useNuxtApp().$logger
+  return useNuxtApp().$logger as pino.Logger
```

Comment:
Check if you even need this anymore, I wouldn't be surprised if $logger was unknown for you because of the mismatched TS version used by Volar before we fixed that